.github/workflows/main.yml

```diff
@@ -1,39 +1,71 @@
 name: tests
 env:
-  CI: true
-  FORCE_COLOR: 1
+    CI: true
+    FORCE_COLOR: 1
 on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+    push:
+        branches: [master]
+    pull_request:
+        branches: [master]
 
 jobs:
-  chromium_linux:
-    name: "Chromium Linux"
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-    - uses: microsoft/playwright-github-action@v1
-    - uses: bahmutov/npm-install@v1
-    - run: yarn run check
-    - uses: codecov/codecov-action@v1
+    chromium_linux:
+        name: "Chromium Linux"
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: microsoft/playwright-github-action@v1
+            - uses: bahmutov/npm-install@v1
+            - run: yarn run check
+            - uses: codecov/codecov-action@v1
 
+    webkit_macos:
+        name: "Webkit Mac"
+        runs-on: macos-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: microsoft/playwright-github-action@v1
+            - uses: bahmutov/npm-install@v1
+            - run: npx playwright-test --runner tape --browser webkit test.js
```

Comment:
```suggestion
            - run: npx playwright-test --runner tape --browser webkit
```

Reply:
Won't `pw-test` pick up `react-native.test.js` this way?

Reply:
ah correct